### ISSUE TITLE
Wolfram is broken

### ISSUE THREAD

**Issue by aaaschmitt** [cs10/Alonzo#92]
https://github.com/strax/node-wolfram

This is currently broken because it relys on limxmljs which has some weird problem in a c++ file. In order to deploy the build I am going to remove wolfram from out node dependencies. node-wolfram should consider relying on xml2js instead, which I think you were looking into @cycomachead

**Comment by cycomachead** [comment]
Yeah, I need to re-write some stuff, but I think I can just upgrade some deps. I have my own forks, but haven't done serious testing.
